### PR TITLE
cmd, daemon, snap, snappy: show purchase information 

### PR DESCRIPTION
**Pull request by pete-woods**
Read information about the logged-in user's purchases from the
purchase server. Present this additional information on the REST
interface and in the output of the "find" command.

### CODE REVIEW

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by pete-woods** [comment]
Stacked on-top of: https://github.com/ubuntu-core/snappy/pull/690/files

File: daemon/api_test.go [hunk 1]
```diff
@@ -202,6 +202,7 @@ func (s *apiSuite) TestSnapInfoOneIntegration(c *check.C) {
 			"update_available":   "v2",
 			"rollback_available": "v0",
 			"channel":            "stable",
+			"price":              float64(0),
```
Comment (zyga) [inline daemon/api_test.go]: Isn't there a decimal type that could be used instead. Using floats for currency is a great way to lose (or make) money.

Comment (pete-woods) [inline daemon/api_test.go]: I guess I could use something like: https://github.com/shopspring/decimal

Fortunately this number is only used for display purposes (comes through as a Javascript numeric type)

**Comment by niemeyer** [comment]
I'm working with Pete to tweak this and get it merged in smaller pieces.

**Pull request closed without merging**